plugins/faqs/enhance.js

```diff
@@ -0,0 +1,41 @@
+function slugify(input) {
```

Comment:
Cant we borrow this from Vuepress?

Reply:
I suppose we can!

https://github.com/vuejs/vuepress/blob/364571bc6155b8bc36bf8b99d22e7b4739e4a05a/packages/%40vuepress/shared-utils/src/slugify.ts

Reply:
I tried and it errors out within their code, I don't quite understand it.

```
VM20402 slugify.js:16 Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
```

Reply:
Most probably... we cant transpile their code I guess. ..